support-workers/src/typescript/services/salesforce.ts

```diff
@@ -92,19 +86,16 @@ export class SalesforceService {
 	createContactRecords = async (
 		user: User,
 		giftRecipient: GiftRecipient | null,
-	): Promise<SalesforceContactRecords> => {
-		const upsertResponse = await this.upsert(
+	): Promise<SalesforceContactRecord> => {
+		const buyerResponse = await this.upsert(
 			createContactRecordRequest(user, giftRecipient),
 		);
-		const recipientUpsertResponse = await this.maybeAddGiftRecipient(
-			upsertResponse.ContactRecord,
+		const giftRecipientResponse = await this.maybeAddGiftRecipient(
```

Comment:
I know it's existing I'd call this maybeUpsertGiftRecipient as then it's clear it's actually calling SF (I was wondering why it was trying to add the gift recipient (to the request) after the upsert had completed.)